franklin/builder/serializers.py

```diff
@@ -14,13 +14,26 @@ class Meta:
             },
         }
 
+        
+class EnvironmentSerializer(serializers.ModelSerializer):
+
+    class Meta:
+        model = Environment
+        fields = ('name', 'url', 'status')
+        extra_kwargs = {
+            "github_id": {
+                "validators": [],
+            },
+        }
+
 
 class SiteSerializer(serializers.ModelSerializer):
     owner = OwnerSerializer()
+    environments = EnvironmentSerializer(many=True, required=False)
```

Comment:
Adding this and setting it to `require=False` Lets us use this serializer is 2 ways:
- When registering a new project for the first time, we don't need/want environment details and just use the site and owner details to initialize everything.
- When the dashboard wishes to list details about active projects, we want it to also fetch details about the environments (url, status, etc.)